bika/lims/browser/sample/view.py

```diff
@@ -380,32 +388,37 @@ def folderitem(self, obj, item, index):
         obj = item['obj']
 
         item['replace']['getSampleID'] = "<a href='%s'>%s</a>" % \
-            (item['url'], obj.getSampleID())
+            (item['url'], obj.getSampleID)
+        analysis_requests_brains = arc(portal_type='AnalysisRequest', UID=obj.getAnalysisRequestsUID)
```

Comment:
I think a best approach would be to add a metadata field for Analysis Request UIDs in Sample catalog. This way, additional query to retrieve the Analysis Requests for the current Sample wouldn't be needed.

Reply:
@xispa I've renamed the column name to ```AnalysisRequestUIDs```. However, as far I know the brain of the analysis request should be retrieved  so as to access the analyisis request UID.

https://github.com/naralabs/bika.lims/blob/54601a6274a3831fa4a9f189778095a71346c3ea/bika/lims/browser/sample/view.py#L393-L399